src/znc.cpp

```diff
@@ -628,6 +629,7 @@ bool CZNC::WriteNewConfig(const CString& sConfigFile) {
     VCString vsLines;
 
     vsLines.push_back(MakeConfigHeader());
+    vsLines.push_back("PidFile = /run/znc/znc.pid");
```

Comment:
Isn't the standard to use `/run/process.pid`?  Why do we have to place it in a sub folder?  We may also wish to make this a config level item which can pull from the conf file if someone needs a customized setup.